### DISCUSSION TITLE
[Vote ended on 2023-02-14] Cancel removal process for google.cloud from Ansible 8 (i.e. keep it in Ansible 8)

### DISCUSSION THREAD

**Discussion by mariolenz** [ansible-community/community-topics#195] [Votes done]
This vote is about **cancelling the removal process of google.cloud from Ansible 8**.

When we finished voting on #105, folks from Google showed up and said that they want to start maintaining the collection again. Since then, there has been work on the collection and also some new releases.

Please vote +1 to cancel the removal process and keep google.cloud in Ansible. Vote -1 to not cancel the removal process, that is remove the collection from Ansible 8. For discussions, please do not use this vote but the original issue.

**Comment by mariolenz** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by jillr** [comment]
+1

**Comment by acozine** [comment]
+1 It seems the process is working well!

**Comment by gotmax23** [comment]
+0

**Comment by russoz** [comment]
+1

**Comment by cybette** [comment]
+1

**Comment by ssbarnea** [comment]
+1
